chsdi/templates/htmlpopup/geotechnik_mineralische_rohstoffe200.mako

```diff
@@ -2,13 +2,19 @@
 
 <%def name="table_body(c,lang)">
 <%
-    PDF_Link = 'https://dav0.bgdi.admin.ch/kogis_web/downloads/geologie/geotechnik/' + c['attributes']['legend'] 
+
+    from chsdi.lib.helpers import resource_exists
+    pdf = None
```

Comment:
I don't think you need to test for the existence of the resource (as it was not the case previously) or if you do, you should use the varaible `pdf` you define in your mako.